common/flatpak-utils.c

```diff
@@ -898,6 +898,10 @@ flatpak_get_allowed_exports (const char     *source_path,
     {
       g_ptr_array_add (allowed_extensions, g_strdup (".desktop"));
     }
+  else if (strcmp (source_path, "share/gnome/autostart") == 0)
+    {
+      g_ptr_array_add (allowed_extensions, g_strdup (".desktop"));
+    }
   else if (flatpak_has_path_prefix (source_path, "share/icons"))
```

Comment:
We should add a kde location here as well. Unfortunately, there's no 'standard' location in /usr/share that works.